src/targets.rs

```diff
@@ -176,6 +189,16 @@ pub trait EspCommon {
     }
 
     fn spi_attach(&self, param: u32) {
+        // Stub calls spi_attach automatically when it boots, therefore, we need to
+        // "unattach" the flash before attaching again with different
```

Comment:
detach as the opposite of attach? (you can ignore me)

Reply:
of course 🤦 I was wondering why `unattach` sounds so weird to me.. thanks!